src/GtkExtensions.jl

```diff
@@ -200,6 +202,23 @@ get_tab(notebook::Gtk.GtkNotebook,page_num::Int) = convert(Gtk.GtkWidget,ccall((
 
 set_tab_label_text(notebook::Gtk.GtkNotebook,child,tab_text) = ccall((:gtk_notebook_set_tab_label_text,Gtk.libgtk),Void,(Ptr{Gtk.GObject},
 Ptr{Gtk.GObject},Ptr{UInt8}),notebook,child,tab_text)
+popup_disble(notebook::Gtk.GtkNotebook) = ccall((:gtk_notebook_popup_disable ,Gtk.libgtk),
+      Void,
+      (Ptr{Gtk.GObject},),
+      notebook)
+function tab_num(notebook::Gtk.GtkNotebook,widget)
+    return ccall((:gtk_notebook_page_num,Gtk.libgtk),
+          Cint,
+          (Ptr{Gtk.GObject},Ptr{Gtk.GObject}),
+          notebook,widget) +1
+end
```

Comment:
This is already in Gtk.jl : https://github.com/JuliaLang/Gtk.jl/blob/40f7442dea20919c5a7b137594fb6d7636fa2329/src/layout.jl#L232

I you want just the +1 you can do tab_num(...) = pagenumber(...) + 1